conf/application.conf

```diff
@@ -13,10 +13,16 @@ play.ws.compressionEnabled=true
 
 google {
   # These are dev credentials, only valid when running on localhost
-  clientId = "CHANGE_ME"
-  clientSecret = "CHANGE_ME"
+  clientId = "1065226601892-jqg9gmbbfa4j6aooqh62bgh1sjfeh5ln.apps.googleusercontent.com"
+  clientSecret = ${?GOOGLE_CLIENT_SECRET}
   redirectUrl = "http://localhost:9000/oauth2callback"
 }
 
 play.application.name="zuora-exporter"
 #play.application.loader="com.gu.cm.ConfigurationGuiceApplicationLoader"
+
+secret.zuora.username="nick.satterly+testDrive@theguardian.com"
```

Comment:
shouldn't put secrets in version control, you should create a file ~/.configuration-magic/zuora-exporter.conf for secrets and put them there.
In prod, configuration magic looks in dynamo for the secrets.  Non secrets should stay here though.